src/features/editing/bpmnModelerBuilder.ts

```diff
@@ -144,7 +144,7 @@ export class BpmnModelerBuilder {
           }
 
           // open diagram
-          openDiagram('${this.contents}');
+          openDiagram(atob('${Buffer.from(this.contents).toString("base64")}'));
```

Comment:
I'm not sure whether we should do that. That would update the full contents of the BPMN file and we can't guarantee that it will work for all occurrences.

Instead of escaping the full content, maybe it's possible to only escape the related `<script></script>` tags?

Reply:
One (hacky) workaround could be to ensure we have a bpmn prefix for script elements when opening the diagram.

```xml
<bpmn:script> 
  def scriptVar = "test123"  
  execution.setVariable("myVar", scriptVar)
</bpmn:script>
 ```